src/Engines/nativeEngine.ts

```diff
@@ -71,8 +71,8 @@ interface INativeEngine {
     setFloat4(uniform: WebGLUniformLocation, x: number, y: number, z: number, w: number): void;
 
     createTexture(): WebGLTexture;
-    loadTexture(texture: WebGLTexture, buffer: ArrayBuffer | ArrayBufferView | Blob, mipMap: boolean, invertY: boolean): void;
-    loadCubeTexture(texture: WebGLTexture, data: Array<Array<ArrayBufferView>>, flipY : boolean): void;
+    loadTexture(texture: WebGLTexture, buffer: ArrayBuffer | ArrayBufferView | Blob, mipMap: boolean, invertY: boolean): any;

```

Comment:
Maybe return a book instead for these?

Reply:
What's a book?

Reply:
Aye, typing on my phone. Bool :)

Reply:
I'd like to be consistent with createIndexBuffer/createVertexBuffer

Reply:
I don't think that makes sense. The create functions create things and thus return the thing it created. The load functions load things into the first parameter :) A boolean return for load to indicate success makes the most sense to me.